frontend/public/components/namespace.jsx

```diff
@@ -273,97 +274,96 @@ const namespacesRowStateToProps = ({ UI }) => ({
 });
 
 const NamespacesTableRow = connect(namespacesRowStateToProps)(
-  withTranslation()(
-    withUserSettingsCompatibility(
+  withTranslation()(({ obj: ns, index, key, style, metrics, t }) => {
+    const [tableColumns, , loaded] = useUserSettingsCompatibility(
       COLUMN_MANAGEMENT_CONFIGMAP_KEY,
       COLUMN_MANAGEMENT_LOCAL_STORAGE_KEY,
-      undefined,
+      null,
```

Comment:
Please keep the default as undefined so that this value are not stored in user settings configmap until the user changes the table config.